.travis.yml

```diff
@@ -8,12 +8,12 @@ services:
 before_install:
   - docker build -f ./Nginx/Dockerfile ./Nginx/ --tag=tinyfish:proxy
   - docker run -d -p 5001:80 tinyfish:proxy
-  - curl -I http://127.0.0.1:5001
```

Comment:
Was hoping to save Travis a few cycles if the proxy fails.  I don't want to put a sleep in the build so I had to move this.